server/schema/mutation.js

```diff
@@ -0,0 +1,25 @@
+const bookMutation = `
+    addBook(title: String!, description: String, published_date: String, author_id: ID): Book
+    updateBook(id: ID!, title: String, description: String, published_date: String, author_id: ID): Book
+    deleteBook(id: ID!): Boolean
+`;
+
+const authorMutation = `
+    addAuthor(name: String!, biography: String, born_date: String): Author
+    updateAuthor(id: ID!, name: String, biography: String, born_date: String): Author
```

Comment:
wouldnt Author be always present? `Author!` ?

Reply:
Yes, it will always be present.